### PR TITLE
Update CSOAccount page

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

This PR includes the following proposed change(s):

Resolves: https://justice.gov.bc.ca/jira/browse/FLA-175
- Update CSOAccount page to display user info.

csoAccountUserInfo

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested locally with Storybook, passing Jest tests.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.js [hunk 1]
```diff
@@ -24,6 +32,14 @@ export default function CSOAccount({ confirmationPopup }) {
     }
   }, [termsAccepted]);
 
+  const icon = (
+    <div style={{ color: "rgb(252,186,25)" }}>
+      <MdPerson size={32} />
+    </div>
+  );
+  const translatedApplicantInfo = translateApplicantInfo(applicantInfo);
+  const applicantTable = <Table elements={translatedApplicantInfo} />;
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.js:41]: nit:
seems simple enough so could just combine

```
const applicantTable = <Table elements={translateApplicantInfo(applicantInfo)} />;
```

File: src/frontend/efiling-frontend/src/components/page/home/Home.js [hunk 2]
```diff
@@ -31,8 +42,21 @@ const checkCSOAccountStatus = (
         const csoAccountExists = userDetails.accounts.some(
           element => element.type === "CSO"
         );
-        if (csoAccountExists) setCsoAccountExists(true);
+        if (csoAccountExists) {
+          setCsoAccountExists(true);
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/components/page/home/Home.js:46]: nit: this change can be undone and can remain on one line

File: src/frontend/efiling-frontend/src/components/page/home/Home.js [hunk 3]
```diff
@@ -16,11 +16,22 @@ export const saveNavigationToSession = ({ cancel, success, error }) => {
   if (error.url) sessionStorage.setItem("errorUrl", error.url);
 };
 
+const addUserInfo = (bceid, firstName, middleName, lastName, email) => {
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/components/page/home/Home.js:19]: change this to:
```
const addUserInfo = (bceID, firstName, middleName, lastName, emailAddress) => {
```
and then below we can just return all key names and no need to specify the value 😸

File: src/frontend/efiling-frontend/src/components/page/home/Home.js [hunk 4]
```diff
@@ -41,14 +65,18 @@ const checkCSOAccountStatus = (
...
-    setShowLoader
-  );
+  useEffect(() => {
+    checkCSOAccountStatus(
+      queryParams.submissionId,
+      setCsoAccountExists,
+      setShowLoader,
+      setApplicantInfo
+    );
+  }, [queryParams.submissionId]);
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/components/page/home/Home.js:79]: question: is `queryParams.submissionId` the only thing we need in this [] for useEffect or do we need to specify others as well?

File: src/frontend/efiling-frontend/src/types/propTypes.js [hunk 5]
```diff
@@ -26,5 +26,12 @@ export const propTypes = {
...
     }).isRequired
-  }).isRequired
+  }).isRequired,
+  applicantInfo: PropTypes.shape({
+    bceID: PropTypes.string.isRequired,
+    firstName: PropTypes.string.isRequired,
+    middleName: PropTypes.string,
+    lastName: PropTypes.string.isRequired,
+    emailAddress: PropTypes.string.isRequired
+  })
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/types/propTypes.js:36]: question: is it not required?

File: src/frontend/shared-components/src/components/display-box/DisplayBox.js [hunk 6]
```diff
@@ -3,14 +3,14 @@ import PropTypes from "prop-types";
 
 import "./DisplayBox.css";
 
-export default function DisplayBox({ styling, icon, element }) {
+export const DisplayBox = ({ styling, icon, element }) => {
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/shared-components/src/components/display-box/DisplayBox.js:6]: 👍

File: src/frontend/efiling-frontend/src/components/page/home/Home.js (hunk 4 above)
Comment (adodge-bcg) [reply src/frontend/efiling-frontend/src/components/page/home/Home.js:79]: This was the only one that came up when running locally.

File: src/frontend/efiling-frontend/src/components/page/home/Home.js [hunk 7]
```diff
@@ -16,11 +16,22 @@ export const saveNavigationToSession = ({ cancel, success, error }) => {
   if (error.url) sessionStorage.setItem("errorUrl", error.url);
 };
 
+const addUserInfo = (bceid, firstName, middleName, lastName, emailAddress) => {
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/components/page/home/Home.js:19]: why not rename this to bceID?

File: src/frontend/efiling-frontend/src/components/page/home/Home.js [hunk 8]
```diff
@@ -16,9 +16,9 @@ export const saveNavigationToSession = ({ cancel, success, error }) => {
   if (error.url) sessionStorage.setItem("errorUrl", error.url);
 };
 
-const addUserInfo = (bceid, firstName, middleName, lastName, emailAddress) => {
+const addUserInfo = (bceID, firstName, middleName, lastName, emailAddress) => {
   return {
-    bceID: bceid,
+    bceID,
```
Comment (sdevalapurkar-bcgov) [inline src/frontend/efiling-frontend/src/components/page/home/Home.js:21]: nice!

**Pull request merged**